Fix images

apreshill opened the pull request:
Phew this was a toughie! I think this should work now though. You can see the final version here:
https://fix-images--lucid-galileo-4b6e4f.netlify.com/

Wait to merge this PR in- I have one more update I'll get to tonight but wanted to give you a heads up :)

apreshill committed:
remove .DS_Store that snuck in

apreshill committed:
suggest post slug without day, and add commented text re Font Awesome version

apreshill committed:
ah all the .DS_Stores

apreshill committed:
change featured post layout

apreshill committed:
add silly featured image to share keys post to test

apreshill committed:
test featured image with do your worst post

apreshill committed:
fix permalinks

apreshill committed:
update post metadata

apreshill committed:
add alt text to featured image

apreshill committed:
update archetypes

apreshill commented:
OK I made it so that if you ever include any image file (it should work for any image type) with the word *featured* in the file name, your theme will now automatically recognize that image as the featured image. No need to enter the featured image file name or path. It assumes you are doing page bundles so don't put the image in `static/`. The post archetypes are updated to reflect this so when you use the new post add in those options will look a bit different. I also:

- fixed your permalinks (your theme is confusing- the name of the actual content folder was `blog/` not `posts/`, so I had to update the toml to get the permalinks to catch). 
- clarified in your `config.toml` that your version of font awesome in this theme is 4.7, so you won't be able to use icons from the font awesome version 5 collection :(
- removed a bunch of those pesky `.DS_Store` files that crept in

Feel free to merge in! I see some more notes in our shared doc- will do separate PRs for those :)

allisonhorst commented:
Whoa, I'm so curious to know how I made such a mess! But more importantly, it wooorks! OK I think it's up and running - that 'featured' in image title trick is amazing...thank you so much for working on this! Looking forward to chatting soon :)

apreshill commented:
No this was not you- your theme wanted the featured images in `static/img` I think (it was hard coded, but unclear unless you actually read the code). So I updated the actual Hugo code so you can use bundles for both your featured image and the post images. Another good example of why I never change anything in the `themes` folder: you can now still update your theme (which is essentially just replacing that folder with the updated folder from the designer's github repo) and the layout I added will still *override* his layout. So again, overriding instead of editing- you now have an `archetypes` folder, which also overrides all the theme's archetypes :)
